RNTester/src/RNTesterPage.re

```diff
@@ -10,67 +10,45 @@ let styles =
       }
     );
 
-module RNTesterPage = {
-  include ReactRe.Component;
-  let name = "RNTesterPage";
-  type props = {
-    noScroll: bool,
-    noSpacer: bool,
-    children: list ReactRe.reactElement,
-    title: option string
-  };
-  let contentWrapper props children =>
-    if props.noScroll {
-      <View key="wrapper" style=styles##wrapper> (ReactRe.listToElement children) </View>
+let component = ReasonReact.statelessComponent "RNTesterPage";
+
+let make ::title=? ::noScroll=false ::noSpacer=false children => {
```

Comment:
Thought, that it's nice with the JSX in the other component. And at least with the new API its not that much difference any more :)